### DISCUSSION TITLE
Difficult to find page to create an API Key

### DISCUSSION THREAD

**Discussion by GregLoring** [simple-login/app#1110] [Q&A]
TL;DR https://app.simplelogin.io/dashboard/api_key

I found SimpleLogin through Bitwarden's generate username functionality.  Bitwarden needed an API Key.  I looked all over SimpleLogin's app and documentation for a place to create API Keys, but I couldn't find one anywhere through the app (I did find the api's POST endpoint in the api docs).  I'm not saying it's not there, just that I couldn't find it.

I finally located it through this 3rd party blog post https://www.ghacks.net/2022/06/02/bitwardens-username-generator-now-supports-simplelogin-anonaddy-and-firefox-relay-email-alias-services/
